inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -237,19 +237,21 @@ def send_and_wait_robotupload(
 ):
     """Get the MARCXML from the model and ship it."""
 
-    @wraps(send_and_wait_robotupload)
+    @wraps(send_robotupload)
     def _send_robotupload(obj, eng):
         from inspirehep.dojson.utils import legacy_export_as_marc
         from inspirehep.utils.robotupload import make_robotupload_marcxml
 
-        combined_callback_url = os.path.join(
-            current_app.config["SERVER_NAME"],
-            callback_url
-        )
-        if not combined_callback_url.startswith('http'):
-            combined_callback_url = "https://{0}".format(
-                combined_callback_url
+        combined_callback_url = ""
```

Comment:
I'm not sure how legacy interprets the callback_url parameter and such... maybe @kaplun can shed some light? (I tried for 5 min to find out where is the code of the batchupload on the old code... no luck).